packages/react-ui-core/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@rentpath/react-ui-core",
-  "version": "18.2.0",
+  "version": "18.2.1",
```

Comment:
There is no reason to bump this here. It's done automatically when we release.

Reply:
ok, gotcha

Reply:
Also, this would be a minor version according to semver
https://semver.org/#spec-item-7